Lib/urllib/request.py

```diff
@@ -2727,35 +2725,21 @@ def proxy_bypass_registry(host):
         if not proxyEnable or not proxyOverride:
             return 0
         # try to make a host list from name and IP address.
```

Comment:
could you refactor the lines below here into its own `def _proxy_bypass_winreg_override(host, override):` function and do a chained call to that `return _proxy_bypass_winreg_override(host, proxyOverride)` from here?

We're appear to currently be lacking unittests for the platform specific proxy override _logic_, this would enable a direct unittest to be written against the _proxy_bypass_winreg_override logic below here in test_urllib.py. (do that as well)

thanks!

Reply:
Understood, I've created the `_proxy_bypass_winreg_override` function. For the test, I've placed it in `test_urllib2.py` to keep it near the macOS proxy bypass test `test_osx_proxy_bypass`.